Dashboard coachee (meetings list) + Nouveau header

JordhanMadec opened the pull request:
Première version du dashboard coachee + nouveau header, le bouton "+"
n'effectue rien pour le moment

JordhanMadec committed:
Dashboard coachee (meetings list) + Nouveau header

Première version du dashboard coachee + nouveau header, le bouton "+"
n'effectue rien pour le moment

guillaumeLeRoy commented on src/app/header/header.component.ts:
```diff
@@ -1,23 +1,24 @@
+import {Coachee} from '../model/coachee';
 
 @Component({
   selector: 'rb-header',
```
désormais le prefix des component n'est plus "rb" mais "eritis"

guillaumeLeRoy commented on src/app/meeting/meeting-list/meeting-list.component.html:
```diff
@@ -1,12 +1,19 @@
+<rb-header></rb-header>
 <div class="container">
-  <h4> Liste de vos meetings</h4>
+  <h4 class="text-right">Bonjour {{(user | async)?.display_name}}, il vous reste <span class="nb-seances">{{(meetings | async)?.length}}</span> séances</h4>
```
je pense que tu le sais mais meetings.length n'est pas la bonne valeur. vais ajouter la notion manquante dans l'api

guillaumeLeRoy commented on src/app/meeting/meeting-list/meeting-list.component.html:
```diff
@@ -1,12 +1,19 @@
+  <h4 class="text-right">Bonjour {{(user | async)?.display_name}}, il vous reste <span class="nb-seances">{{(meetings | async)?.length}}</span> séances</h4>
+  <p class="text-right">
+    Cliquez ici pour planifier une nouvelle séance
+    <a class="btn-floating btn-large waves-effect waves-light add-meeting-btn" (click)="goToMeetings()">
```
"goToMeetings()" est incorrect puisque le user doit être redirigé vers le component permettant de donner ses dispo

guillaumeLeRoy commented on src/styles.css:
```diff
@@ -1,25 +1,32 @@
+  --main-blue-filter: rgba(35, 88, 128, 0.6);
 }
 
 .recipes_grid {
```
A virer je pense

JordhanMadec committed:
Choix des plages de disponniblités

Quelques petits bugs sont à corriger + les boutons d'édition et
"terminé" ne sont pas fonctionnels

guillaumeLeRoy committed:
add contract plans to signup

Commit:
Add Contract Plan to sign up AND plan to Coachee model object

guillaumeLeRoy committed:
Merge pull request #5 from guillaumeLeRoy/feature/add_contract_plan_to_coachee_signup

Feature/add contract plan to coachee signup

JordhanMadec committed:
"Can't find contarctPlanId" bug fixed

JordhanMadec committed:
Worflow updated

Modifiaction du worflow
Revoir la notion de sélection d'un créneau horaire

guillaumeLeRoy committed:
Create a meeting before setup any potential dates

guillaumeLeRoy commented on src/app/meeting/meeting-date/meeting-date.component.ts:
```diff
@@ -18,34 +18,52 @@ export class MeetingDateComponent implements OnInit, OnDestroy {
   @Output()
   potentialDatePosted = new EventEmitter<MeetingDate>();
```
pour qui est-ce que tu veux émettre des events ?
je pense que ce que tu veux faire c'est updater la liste qd un nouvel élément est ajouté, par besoin d'eventEmitter pr ça, juste un array et l'update se fera automatiquement

JordhanMadec replied:
C'était déjà là ça, je ne l'ai pas rajouté mais en effet je comprenais pas trop quand ça servait

JordhanMadec committed:
meetings grid v1

guillaumeLeRoy committed:
Merge branch 'develop' into feature/dashboard_coachee

# Conflicts:
#	src/app/app.module.ts

JordhanMadec committed:
Meetings grid v2

JordhanMadec committed:
Merge remote-tracking branch 'origin/feature/dashboard_coachee' into feature/dashboard_coachee

guillaumeLeRoy committed:
connect to API : update & delete potential date

guillaumeLeRoy committed:
Merge branch 'feature/dashboard_coachee' of https://github.com/guillaumeLeRoy/recipe-book into feature/dashboard_coachee

# Conflicts:
#	src/app/meeting/meeting-date/meeting-date.component.ts

JordhanMadec committed:
Meeting card review

guillaumeLeRoy committed:
Add post meeting review

guillaumeLeRoy committed:
Add post meeting review

guillaumeLeRoy committed:
Merge branch 'feature/dashboard_coachee' of https://github.com/guillaumeLeRoy/recipe-book into feature/dashboard_coachee

guillaumeLeRoy committed:
Add pre-meeting component to Calendar page

JordhanMadec committed:
Improve calendar refreshing

guillaumeLeRoy committed:
modify pre session component

guillaumeLeRoy committed:
Merge branch 'feature/dashboard_coachee' of https://github.com/guillaumeLeRoy/recipe-book into feature/dashboard_coachee

# Conflicts:
#	src/app/meeting/meeting-date/meeting-date.component.html
#	src/app/meeting/meeting-date/meeting-date.component.ts

guillaumeLeRoy committed:
Functional goal and context

JordhanMadec committed:
Improved date modifications

JordhanMadec committed:
Resolved confilcts

JordhanMadec committed:
Merge remote-tracking branch 'origin/feature/dashboard_coachee' into feature/dashboard_coachee

# Conflicts:
#	src/app/meeting/meeting-date/meeting-date.component.html
#	src/app/meeting/pre-meeting.component.html
#	src/app/meeting/pre-meeting.component.ts

JordhanMadec committed:
Fixed hours format + slider modificiation hour

guillaumeLeRoy committed:
Display available sessions count & fix alignment for « Terminer » button

guillaumeLeRoy committed:
Change « availableSessionsCount »

JordhanMadec committed:
Dashboard card_version

JordhanMadec committed:
Merge remote-tracking branch 'origin/feature/dashboard_coachee' into feature/dashboard_coachee

# Conflicts:
#	src/app/meeting/meeting-date/meeting-date.component.html
